docs/reference/errors-and-warnings/NU1110.md

```diff
@@ -0,0 +1,36 @@
+---
+title: NuGet Error NU1110
+description: NU1110 error code
+author: donnie-msft
+ms.author: eagoodso
+ms.date: 10/27/2023
+ms.topic: reference
+f1_keywords: 
+  - "NU1110"
+---
+
+# NuGet Error NU1110
+
+<pre>NU1110: The package `B` is available in the Global packages folder, but the source it came from `https://api.nuget.org/v3/index.json` is not one of the configured sources.
+</pre>
+
+### Issue
+
+A previous restore retrieved package `B` from a package source which your current solution does not have configured.
+
+While performing a package installation of package `A`, a dependent package `B` was needed transitively to complete the action. Restore found `B` in your [Global packages folder](../../consume-packages/managing-the-global-packages-and-cache-folders.md), and the accompanying `nupkg.metadata` indicates a `Source` value which it cannot find in your solution.
+
+When [package source mapping](../../../consume-packages/package-source-mapping.md) is enabled, NuGet attempts to map transitive dependencies automatically by analyzing the source in the Global packages folder.
+
+### Solution
+
+#### Option 1. 
+Create a package source mapping for package `B`.
+
+#### Option 2.
+Clear package `B` from your Global packages folder so that restore can download from one of your configured packages sources. For more information, see [Managing the global packages, cache, and temp folders](../../../consume-packages/managing-the-global-packages-and-cache-folders.md).
```

Comment:
```suggestion
Clear package `B` from your Global packages folder so that restore can download from one of your configured packages sources. For more information, see [Managing the global packages, cache, and temp folders](../../consume-packages/managing-the-global-packages-and-cache-folders.md).
```

Reply:
First time, I had the right path. I then added `.md` but accidentally left my test of 3 levels up. Trying again..